code/web/.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules/
 build/
+coverage/
```

Comment:
Awesome that coverage has been added, and also that you remembered to add it to .gitignore files!

Reply:
Ha! Massive thanks to Rostam for that one.

Reply:
thanks!